modules/@apostrophecms/schema/index.js

```diff
@@ -1427,6 +1425,9 @@ module.exports = {
             try {
               await convert(req, field, data, object);
             } catch (e) {
+              if (e instanceof Error) {
```

Comment:
If we get an `Error` send it on.

Reply:
This would send on pretty much everything, because self.apos.error does construct an Error.

Reply:
Not sure what you're trying to do.

Reply:
The problem was that an error thrown in the field's `convert` method was buried here, including the helpful message. I couldn't see the `message` property on it by logging, but I see it's there now to check and use.